test/browser_tests/spec_suites/careers-application-process.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+var ApplicationProcess = require(
+    '../page_objects/page_careers-application-process.js'
+  );
+
+describe( 'The Application Process Page', function() {
+  var page;
+
+  beforeAll( function() {
+    page = new ApplicationProcess();
+    page.get();
+  } );
+
+  it( 'should properly load in a browser', function() {
+    expect( page.pageTitle() ).toBe( 'Job Application Process' );
+  } );
+
+  it( 'should have a sideNav', function() {
+    expect( page.sideNav.isPresent() ).toBe( true );
+  } );
+
+  it( 'should have an intro title', function() {
+    expect( page.introSectionTitle.getText() )
+    .toBe( 'Job Application Process' );
+  } );
+
+  it( 'should have three job application interfaces ', function() {
+    expect( page.jobApplicationsInterfaces.count() ).toEqual( 3 );
+  } );
+
+  it( 'should have an ethics link ', function() {
+    expect( page.ethicsLink.isPresent() ).toBe( true );
+  } );
+
+  it( 'should have a career info section', function() {
+    var infoSectionTitles =
+    [ 'Current Openings', 'Working at the CFPB',
+    'Students & Recent Graduates' ];
+    var infoSectionLinks =
```

Comment:
Should the logic here go into the page object so that this could look something like:

```
var infoSectionLinks = page.infoSectionLinks( [ '/careers/current-openings/', 
                                                                           '/careers/working-at-cfpb/',
                                                                           '/careers/students-and-graduates/' ]  );
```

?